### ISSUE TITLE
[a11y-defect-x]: Issue description

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#119982]
### Point of contact

Sarah Horton

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

...

### Reproduction steps

1.
2.
3.
...

### Proposed solution or next steps

...

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [x] Keyboard
- [ ] Voice command
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else
